Makefile

```diff
@@ -0,0 +1,51 @@
+GO ?= $(shell command -v go 2> /dev/null)
+MATTERMOST_CLOUD_IMAGE ?= mattermost/mattermost-cloud:test
+MACHINE = $(shell uname -m)
+BUILD_IMAGE = golang:1.12
```

Comment:
Could we call this something like `DOCKER_BUILD_IMAGE` and `DOCKER_BASE_IMAGE`? Their top-level nature suggests they apply to the project as a whole, so it would be good to clarify.